jsx/Form.js

```diff
@@ -1459,6 +1459,34 @@ ButtonElement.defaultProps = {
   },
 };
 
+/**
+ * CTA component
+ * React wrapper for <button> element, typically used to submit forms
+ */
+class CTA extends Component {
```

Comment:
@zaliqarosli Comments?

Reply:
i am in favour of this component. right now, our ButtonElement has divs around it with css that makes using it as a stand alone button difficult to format. However, ButtonElement is still useful when used inside a FormElement (because of grid formatting). our renderActions method in DataTable.js currently renders <button> instead of our ButtonElement for every CTA. This new CTA component can be implemented here to replace <button>. I do suggest changing the doc comment to say that this component should only be used for stand alone call to actions that aren't typically in forms. ButtonElements should still be used for forms i.e. menu filter. CTAs should open up modal window or redirect to a form for example. the doc comment should also say 'call to action' to explain the acronym.